vector/src/main/java/im/vector/app/features/login/terms/PolicyItem.kt

```diff
@@ -28,7 +28,7 @@ import im.vector.app.core.epoxy.VectorEpoxyHolder
 import im.vector.app.core.epoxy.onClick
 import im.vector.app.core.extensions.setHorizontalPadding
 
-@EpoxyModelClass(layout = R.layout.item_policy)
+@EpoxyModelClass
```

Comment:
Strange that this class does not extend `VectorEpoxyModel` like the other. Maybe a small mistake from me. If there is no good reason not to do it, maybe an opportunity to fix that...

Reply:
ah great catch, will fix! this must have been one of the first items I updated before deciding it would be better to provide the id via the constructor